dist/stat_data.py

```diff
@@ -245,6 +264,25 @@ def __init__(self, name, desc, flags=''):
     DhandleStat('dh_sweeps', 'connection sweeps'),
 
     ##########################################
+    # Locking statistics
+    ##########################################
```

Comment:
I don't have an opinion; I'm happy to do the work if @sueloverso and @agorrod identify which locks should be tracked.